apps/system/js/remote/message_controller.js

```diff
@@ -54,15 +54,16 @@
         ', ' + JSON.stringify(data.detail));
 
       switch(data.type) {
-        case 'launch-presentation-app':
-          this.debug('request to launchPresentationApp');
+        case 'app-config-ready':
           Service.request('launchPresentationApp', data.detail)
             .then(() => {
-              this.postMessage('launch-app-success', {
+              this.publish('launch-app-success', {
```

Comment:
I'm just curious who is the expecting receiver of the event `launch-app-success` and `launch-app-error`.